tests/unit/client.py

```diff
@@ -326,12 +326,12 @@ def test_node_connect_network_error(self):
 
     def test_node_detach_network(self):
         C.node.connect_network('node-04', 'eth0', 'net-04', 'vlan/native')
-        time.sleep(1)
+        time.sleep(2)  # To avoid a potential race condition
         assert C.node.detach_network('node-04', 'eth0', 'net-04') is None
 
     def test_node_detach_network_error(self):
         C.node.connect_network('node-04', 'eth0', 'net-04', 'vlan/native')
-        time.sleep(1)
+        time.sleep(2)  # To avoid a potential race condition
```

Comment:
Clarification: Nothing you pass to sleep is going to eliminate the race condition. What I'm looking for is (1) a value that's big enough that failures will be rare, and (2) a comment that calls attention to the fact that there *is* a race condition. This seems to imply that the race condition is elimnated by the call to sleep, which is not true.